examples/app/utils_mfem.hpp

```diff
@@ -43,6 +43,7 @@ using dt3 = mfem::DeviceTensor<3, T>;
 
 
 template <typename Tin, typename Tout>
+__attribute__((annotate("@critical_path(pointcut='around')")))
```

Comment:
Why not using the MACRO here?

Reply:
The macro was causing error here. I think its scope is within the src directory, though I could be wrong.